README.md

```diff
@@ -1,2 +1,24 @@
 # fpp-202y
-Work files for proposed Fortran 202y preprocessor standardization
+
+This repository holds work files for the proposed Fortran 202y preprocessor standardization effort 
+([WG5 work item US-10](wg5-papers/N2234%20Work%20items%20from%20the%20hybrid%20wg5%20meeting%202024-06.txt#L54)).
+
+Relevant published papers appear in the [j3-papers](j3-papers) directory.
```

Comment:
We're moving papers into the `j3-papers` directory at the beginning of the J3 meeting or the end?

P.S. I know there was some discussion about not wanting to move/rename the draft to make it easier to merge in changes for an `r1`, but as long as we have a single commit that moves the file from the old name/location to the new name/location, git should be able to track the move.